app/services/katello/pulp3/pulp_content_unit.rb

```diff
@@ -116,30 +116,28 @@ def self.fetch_content_list(page_opts)
         content_unit_list page_opts
       end
 
-      # rubocop:disable Lint/UselessAssignment
       def self.find_duplicate_unit(repository, unit_type_id, file, checksum)
-        filter_label = 'sha256'
+        filter_label = :sha256
         if unit_type_id == 'ostree_ref'
-          filter_label = 'checksum'
+          filter_label = :checksum
         end
         content_backend_service = SmartProxy.pulp_primary.content_service(unit_type_id)
         duplicates_allowed = ::Katello::RepositoryTypeManager.find_content_type(unit_type_id).try(:duplicates_allowed)
         if repository.generic? && duplicates_allowed
           filename_key = ::Katello::RepositoryTypeManager.find_content_type(unit_type_id).filename_key
           duplicate_sha_path_content_list = content_backend_service.content_api(repository.repository_type, unit_type_id).list(
-            filter_label: checksum,
+            filter_label => checksum,
```

Comment:
i really do not like the ":" style hashes for this reason!  Still not sure why ruby added it